src/App.jsx

```diff
@@ -14,33 +15,87 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
+  let visibleGoods = [...goodsFromServer].sort((good1, good2) => {
+    switch (sortField) {
+      case 'sortAlphabetically':
+        return good1[0].localeCompare(good2[0]);
```

Comment:
The current sorting logic for 'sortAlphabetically' only compares the first character of each item. This will not sort the items alphabetically as expected. Consider using `localeCompare` on the entire string instead: `return good1.localeCompare(good2);`